src/components/new-member/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import Image from "next/image";
+
+const NewMembersCard = ({
+  newMemberName,
+  newMemberImageSrc,
+}: {
+  newMemberName: string;
```

Comment:
Write type separately, like this
```ts
type NewMembersCardProps = {
newMemberName: string;
newMemberImageSrc: string;
}
```
Then can use like this
```tsx
const NewMembersCard = ({
  newMemberName,
  newMemberImageSrc,
}:NewMembersCardProps
```

Reply:
done